core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -464,7 +477,32 @@ object JarCreationTest extends Logging {
       var exception: String = null
       try {
         Class.forName("SparkSubmitClassA", true, Thread.currentThread().getContextClassLoader)
-        Class.forName("SparkSubmitClassA", true, Thread.currentThread().getContextClassLoader)
+        Class.forName("SparkSubmitClassB", true, Thread.currentThread().getContextClassLoader)
+      } catch {
+        case t: Throwable =>
+          exception = t + "\n" + t.getStackTraceString
```

Comment:
I think there might be a function in `Utils` for formatting exceptions as nice human-readable strings.